src/Microsoft.IIS.WebManager/app/webserver/url-rewrite/outbound-rules/outbound-rules.component.ts

```diff
@@ -10,7 +10,7 @@ import { OutboundSection, OutboundRule, PatternSyntax, OutboundTags, ActionType,
     selector: 'outbound-rules',
     template: `
         <error [error]="_service.outboundError"></error>
-        <div *ngIf="!_service.outboundError && _settings">
+        <div *ngIf="!_service.outboundError && _settings != null">
```

Comment:
If we're using _settings != null shouldn't we also use _service.outboundError == null?